setup.py

```diff
@@ -387,6 +387,7 @@ def run(self):
         'sympy>=1.0',
         'numpy>=1.10.4',
         'IPython>=1.0',
+        'pytest>=5.2',
```

Comment:
Wait, why is this getting added here? Is pytest now necessary to import yt? We don’t have nose here already, for example.

Reply:
I think Matt may have been referring to `tests/test_requirements.txt` and `tests/test_minimal_requirements.txt`. Appveyor doesn’t use that so you’ll have to update appveyor.yml seperately. I don’t think this PR should add a new runtime dependency to yt, only a test dependency.

Reply:
Oh, my bad, I'll fix that!